galsim/chromatic.py

```diff
@@ -140,9 +355,14 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
 
         The task of drawImage() in a chromatic context is to integrate a chromatic surface
         brightness profile multiplied by the throughput of `bandpass`, over the wavelength interval
-        indicated by `bandpass`.
-
-        Several integrators are available in galsim.integ to do this integration.  By default,
+        indicated by `bandpass`.  This integration will take place either via brute-force drawing
+        the image at each wavelength, or via an optimized version of the routine that uses some
```

Comment:
I simplified this a bit in https://github.com/GalSim-developers/GalSim/commit/d63eea2637b6a74f2d75048da48f7485d549d400, but please see what you think.